preprocessing/meshing/cube_c/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+cmake_minimum_required(VERSION 3.10)
+
+# set the project name
+project(cube_c)
+
+# specify the C++ standard
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+cmake_policy(SET CMP0074 NEW)
```

Comment:
This line does not make sense here. The policy is only supported for Cmake version > 3.12
See: https://cmake.org/cmake/help/latest/policy/CMP0074.html
I also think that we should provide a quick comment in these cases. I know that we do not do this currently but it would make things a bit clearer. What do you think?